back/src/Infrastructure/Controller/CreatePostController.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Infrastructure\Controller;
+use App\Application\CreatePost\CreatePostCommand;
+use App\Application\CreatePost\CreatePostHandler;
+use App\Infrastructure\Validator\CreatePost\CreatePostRequestValidator;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class CreatePostController extends AbstractController
+{
+
+    public function __construct(
+        private readonly CreatePostHandler $handler,
+        private readonly CreatePostRequestValidator $validator
+    ){}
+
+    public function create(Request $request):JsonResponse
+    {
+
+        $data = $this->validator->validate($request);
```

Comment:
Validation should be in your handler, domain logic should never be in the controller